aten/src/ATen/native/cpu/FusedAdamKernel.cpp

```diff
@@ -233,13 +224,11 @@ typename std::enable_if<
     param_vec = param_vec + Vec(scalar_t(-step_size)) * exp_avg_vec / denom_vec;
     param_vec.store(param_ptr + d);
   }
-  scalar_t grad_val_to_store;
   for (; d < size; d++) {
     scalar_t grad_val = grad_ptr[d];
     if (grad_scale_ptr) {
       grad_val = grad_ptr[d] / scalar_t(*grad_scale_ptr);
```

Comment:
huh now i'm confused why we ever needed the two lines before this at all.

Reply:
Forgot to remove